### ISSUE TITLE
Add unbound custom action to record without reporting

### ISSUE THREAD

**Issue by jprykiel** [jcsteh/osara#963]
Hi.
The Osara configuration dialogue has a checkbox that says report transport state, play, record, et cetera. 
If unchecked, none of these are spoken by the screenreader, if checked, everything is. 
I would like 3 choices instead of two, off, on except record, and on. The goal of this is to have record not being spoken when engaged, so that a nearby microphone won't pick it up, but still have play/stop spoken. 
I would find this useful in my workflow.
Cheers,
JPR

**Comment by jcsteh** [comment]
One potentially confusing thing about this is that if you have reporting of record disabled, it won't say anything when you press record, but it will say stop when you press stop. That feels a bit asymmetric to me.

Can you help me understand the workflow a little better here? I guess your speech is coming through speakers or the like, so the microphone is picking up the speech when it says record? But you still find it useful to have OSARA tell you when you press play?

If we don't end up adding a configuration option for this, another way you could manage it is to have a custom action for record containing mute next message and record.

**Comment by jprykiel** [comment]
Hi Jamie. I would use this mostly when punching on-the-fly, which I do a lot. So I would typically toggle record on and off by pressing the letter r. You're right in saying this is asymmetrical, but not really if you consider toggling record. The stop key would not be picked up by the microphone either. Anyway, thank you for the mute next message option, I might use this one, but I still think this extra Osara config option would make sense.
Cheers,
JPR

**Comment by jcsteh** [comment]
It's currently super difficult to have anything other than check boxes in that dialog, so we'd need to split Report transport state into two separate options: Report play/stop and Report record or similar.

@ScottChesworth , what are your thoughts on this? The asymmetry bothers me a little, but probably not enough that it's a hill I'm willing to die on. It's pretty easy to implement two check boxes here.

**Comment by ScottChesworth** [comment]
A really simple custom action could handle this. I'd prefer to pop that in
the key map unbound, call it something like Record without Reporting. It'd
be available for anyone who needs it and unobtrusive for anyone who doesn't.

**Comment by jprykiel** [comment]
I suppose record without recording might confuse some users. Why not record without announcement, or record without speech?
Anyway, I added   mute next message to the record action and I’m perfectly OK with that
@ScottChesworth were you thinking of another custom action than this?
Cheers,
JPR

**Comment by ScottChesworth** [comment]
Yeah was thinking the same custom action you've made there handles this
fine. It's easy to include that, unbound by default.

**Comment by ScottChesworth** [comment]
JPR has a solution and nobody else has mentioned needing this so I'm closing. If requested in future, I can make a custom action available on The Hoard now there's open access to that resource.
